SPECS/python-looseversion/python-looseversion.spec

```diff
@@ -0,0 +1,70 @@
+%global _description %{expand:
+A backwards/forwards-compatible fork of distutils.version.LooseVersion, for
+times when PEP-440 isnt what you need.
+
+The goal of this package is to be a drop-in replacement for the original
+LooseVersion. It implements an identical interface and comparison logic to
+LooseVersion. The only major change is that a looseversion.LooseVersion is
+comparable to a distutils.version.LooseVersion, which means tools should not
+need to worry whether all dependencies that use LooseVersion have migrated.
+
+If you are simply comparing versions of Python packages, consider moving to
+packaging.version.Version, which follows PEP-440. LooseVersion is better suited
+to interacting with heterogeneous version schemes that do not follow PEP-440.}
+
+Name:           python-looseversion
+Version:        1.3.0
+Release:        1%{?dist}
+Summary:        Version numbering for anarchists and software realists
+
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+
+License:        PSF-2.0
+URL:            https://pypi.org/pypi/looseversion
+Source0:        https://github.com/effigies/looseversion/archive/refs/tags/%{version}.tar.gz#/python-looseversion-%{version}.tar.gz
+
+BuildArch:      noarch
+
+%description %_description
+%package -n python3-looseversion
+Summary:        %{summary}
+BuildRequires:  python3-devel
+BuildRequires:  python3-pytest
```

Comment:
Probably can be surrounded by `with_check` - may speed up when the packages gets queued for building but not a significant concern.

Reply:
Oops. I'll add the conditional.

Reply:
Thanks. The "Also, every now and then, helps to understand the history of the package." part was supposed to go to a different comment but I see we got it right.:P